src/index.scss

```diff
@@ -0,0 +1,66 @@
+ 
+// Taken from @blueprintjs/core/lib/css/blueprint.css 
+@charset "UTF-8";
+
+html {
+  -webkit-box-sizing:border-box;
+          box-sizing:border-box;
+}
+
+*, *::before, *::after {
+  -webkit-box-sizing:inherit;
+          box-sizing:inherit;
+}
+
+body {
+  font-size:14px;
+  font-weight:400;
+  letter-spacing:0;
+  line-height:1.28581;
+  text-transform:none;
+  color:#182026;
+  // font-family:-apple-system, "BlinkMacSystemFont", "Segoe UI", "Roboto", "Oxygen", "Ubuntu", "Cantarell", "Open Sans", "Helvetica Neue", "Icons16", sans-serif;
+}
+
+p {
+  margin-bottom:10px;
+  margin-top:0;
+}
+
+::-moz-selection {
+  background:rgba(125, 188, 255, 0.6);
+}
```

Comment:
```suggestion
```
I think we can probably drop this at this point (https://caniuse.com/?search=%3A%3Aselection).